Sources/Utility/Git.swift

```diff
@@ -22,15 +22,18 @@ extension Version {
 }
 
 public class Git {
-    /// Compute the version -> tag mapping from a list of input `tags`.
-    public static func convertTagsToVersionMap(_ tags: [String]) -> [Version: String] {
+    /// Compute the version -> tags mapping from a list of input `tags`.
+    public static func convertTagsToVersionMap(_ tags: [String]) -> [Version: (normal: String?, vPrefixed: String?)] {
```

Comment:
Instead of returning a tuple here, I think we should return an array of String. The array will have a minimum of one element and maximum of two. It is a little type unsafe to do that but shouldn't really be a problem in practice.

Reply:
Ok. That’s actually what I tried first, but thought it might not be clear what was happening plus it was easier to tell which tag had the ‘v’-prefix this way. It seems much simpler to use arrays now that I think of it!